src/lib.rs

```diff
@@ -1481,3 +1482,30 @@ macro_rules! impl_tuple_zeroable {
 }
 
 impl_tuple_zeroable!(A, B, C, D, E, F, G, H, I, J);
+
+/// Trait for creating a [PinInitialized](PinInit) wrapper containing `T`.
```

Comment:
Could you try to extend the docs a bit more? To me this trait is meant for structs that contain a single value of type `T` that is structurally pinned. (it also might contain other fields)

Reply:
I've extended it and added a small doctest exaple usage.